hydrosdk/exceptions.py

```diff
@@ -1,3 +1,21 @@
+class BadRequest(BaseException):
+    """
+    Exception for client-side errors.
+    """
+    pass
+
+
+class BadResponse(BaseException):
```

Comment:
Python [documentation](https://docs.python.org/3/library/exceptions.html) states that 
>The built-in exception classes can be subclassed to define new exceptions; programmers are encouraged to derive new exceptions from the `Exception` class or one of its subclasses, and **not from** `BaseException`. More information on defining exceptions is available in the Python Tutorial under User-defined Exceptions.

I suggest following this guideline